templates/terraform/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "project" {
+  default = "{{ .Config.Name }}"
+  description = "The name of the project, mostly for tagging"
+}
+
+variable "region" {
+  default = "{{ .Config.Infrastructure.AWS.Region }}"
+  description = "The AWS region"
+}
+
+# {{ if .Config.Infrastructure.AWS.Terraform.RemoteState }}
```

Comment:
I think this portion might be out of date after the last PR merge and we should be able to remove this.

Reply:
I think you missed something about the refactor I did.. Let's chat?